apps/designer/app/shared/props-utils/update-props.ts

```diff
@@ -4,12 +4,13 @@ import {
 } from "@webstudio-is/react-sdk";
 
 export const updateAllUserPropsMutable = (
+  treeId: string,
   allUserProps: AllUserProps,
-  { instanceId, propsId, treeId, updates }: UserPropsUpdates
+  { instanceId, propsId, updates }: UserPropsUpdates
 ) => {
   if (instanceId in allUserProps === false) {
     allUserProps[instanceId] = {
-      id: propsId,
+      id: propsId ?? crypto.randomUUID(),
```

Comment:
We use `import { v4 as uuid } from "uuid";` in other places to generate UUIDs. Not sure which is better, but we should use one or the other everywhere consistently

Reply:
We just talked with Ivan about it. randomUUID is supported everywhere though safari only since 15. So we could migrate if that's ok.

Reply:
checked, what we use is exactly that + fallback https://github.com/uuidjs/uuid/blob/main/src/v4.js

turns out support is really great https://caniuse.com/?search=randomUUID

@TrySound can you switch all of them over? this or separate PR doesn't matter

Reply:
actually only makes sense to switch if native version works in all environments where we might use it: node, workers, browsers

Reply:
Looks like not supported by node globally. Only with crypto module. Though added to cloudflare last year. Will switch later then.

Reply:
So you would go with uuid package then?

Reply:
Yes for now. We can wait node 20 or switch to simple polyfill later.